src/main/java/Duke.java

```diff
@@ -1,10 +1,239 @@
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+// A-Enums: Set Enums for Task type
+enum TASK_TYPE {
```

Comment:
Perhaps name this enum using PascalCase.

```suggestion
enum TaskType {
```